### ISSUE TITLE
[Feature] disable recursive rom search

### ISSUE THREAD

**Issue by RJSzynal** [Aloshi/EmulationStation#314]
It would be useful if it were possible to disable recursion for rom directory listing (whether globally or on a per console basis). In the case of multi-disc PSX games you can create an m3u playlist in place of the rom and put the multiple disc files into a folder but the folder shows in the rom list which is messy and annoying for me.

I've no idea how easy or difficult this is but I thought I'd at least throw it out there in case others would also like to see the same feature.

Thanks for the awesome work, I've been hoping for something like this for years.

**Comment by nilsbyte** [comment]
@nooryani84 Thats a nice work-around, alternatively you simply could remove `.CUE .cue` from the system in `es_systems.cfg` and add `.M3U .m3u`. 

@RJSzynal Try to name your folders containing the actual disc images `.game`(with a period before the name). This should hide it from EmulationStation.

**Comment by nooryani84** [comment]
@nilsbyte Yes that would work, though that would only be helpful for multi-disc games.   I think alternatively it would be good to add an identifier to the filename that perhaps the user defines in es_systems.cfg.  Something that indicates the image is multi-part and should have a sub-tree?   I know this is a lot easier to describe than to implement :)

Perhaps a filter that would allow the user to add .multi to the folder name for games with multiple discs?

**Comment by RJSzynal** [comment]
I've tried the hidden folder fix (`.game`) but it just leaves the folder with no name (i.e. a blank selectable space) in the game list and still has the individual discs inside. I haven't tried also hiding the individual files too but it's becoming much less manageable.
I'm on a fresh install on an RPi2 running RetroPie with the unstable branch of emulationstation.

Also, regarding the "remove `.CUE .cue` from the system in `es_systems.cfg`", you would then have to create .m3u files for all the single disc games too.

Is the solution in my second comment not workable? It seems the neatest solution.

**Comment by nooryani84** [comment]
@RJSzynal The way I got it to work was without removing anything from es_systems.cfg

This way I only need to create an M3U file for multi-disc games.   Remove the .cue extension from the file and instead load them via the M3U.
